src/common/UserAvatar.js

```diff
@@ -53,6 +53,6 @@ const UserAvatar = function UserAvatar(props: Props) {
       </Touchable>
     </View>
   );
-};
+}
 
 export default UserAvatar;
```

Comment:
How about putting this directly on the function definition, as we typically do where applicable for class components? So:
```js
export default function UserAvatar(props: Props) {
```

That seems just a bit simpler still.